### ISSUE TITLE
large strings

### ISSUE THREAD

**Issue by roihe** [kelindar/column#94]
if i do a table.Insert(func(r column.Row) error { r.SetString("name", "Alice"+RandStringBytes(70000)) return nil 	}) it results in a "runtime error: index out of range [75901] with length 70009".
Is there a limit for large strings? Thank you

**Comment by kelindar** [comment]
Right, it's currently limited to 65K. It's probably not the best idea to store large strings in column anyway, not really designed for this at the moment.
